File: app/src/main/java/com/luo/tangramdemo/MainActivity.java
package com.luo.tangramdemo;

import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import android.view.View;

import com.luo.tangramdemo.tangram.TangramActivity;
import com.luo.tangramdemo.virtualview.VirtualViewActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        findViewById(R.id.tv_tangram).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, TangramActivity.class));
            }
        });
        findViewById(R.id.tv_virtual_view).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, VirtualViewActivity.class));
            }
        });
    }
}

File: app/src/main/java/com/luo/tangramdemo/MainActivity.kt
package com.luo.tangramdemo

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.luo.tangramdemo.tangram.TangramActivity
import com.luo.tangramdemo.virtualview.VirtualViewActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        findViewById<View>(R.id.tv_tangram).setOnClickListener { startActivity(Intent(this@MainActivity, TangramActivity::class.java)) }
        findViewById<View>(R.id.tv_virtual_view).setOnClickListener { startActivity(Intent(this@MainActivity, VirtualViewActivity::class.java)) }
    }
}